homeassistant/components/media_player/plex.py

```diff
@@ -457,6 +474,13 @@ def force_idle(self):
         self._session = None
         self._clear_media_details()
 
+    async def async_added_to_hass(self):
+        """Callback when entity is added to hass."""
+        if PLEX_DATA not in self.hass.data:
+            self.hass.data[PLEX_DATA] = PlexData()
```

Comment:
This is not possible as setup_platform already does this.

Reply:
I thought that was what was happening but wasnt 100% sure